it-IT.Rmd

```diff
@@ -97,18 +96,29 @@ Il nostro gruppo sarà felice di aiutarti e di offrirti il supporto necessario (
 
+#### _Mediatore/Mediatrice_
+
+Capiamo che tu possa volere una persona non collegata a Bioconductor che superviosioni la risoluzione del tuo conflitto. Per facilitare ciò, la Commissione del Codice di Comportamento nominerà un/una mediatore/mediatrice da una comunità open-source differente rispetto ai membri del progetto di Bioconductor. Mentre loro non hanno il potere di stabilire, cambiare o impostare decisione prese dalla Commissione del Codice di Comportamento, possono assistere i membri della comunità offrendo loro consultazione, mediazione, istruzione e riferimenti ad altre risorse. Il/La mediatore/mediatrice possono essere contattati direttamente via mail per assicurare un processo confidenziale, imparziale, informale e indipendente.
+
+#### _Procedura dell'incidente_
+
+La Commissione del Codice di Comportamento segue una procedura standard per trattare i conflitti.
+Questa procedura è spigata brevemente [qui](https://bioconductor.github.io/CodeOfConduct/incident_response.html). Le procedure d'incidente sono basate sul libro "How to Respond to Code of Conduct Reports" di Valerie Aurora e Mary Gardiner.
+
 ###  Commissione del Codice di Buon Comportamento
 
-* (Presidente) Saskia Freytag <sas.freytag@gmail.com> (fino al 2023)
-* Yagoub Adam <yagoub@alumni.usp.br> (fino al 2022)
-* Leonardo Collado-Torres <lcolladotor@gmail.com> (fino al 2022)
-* Laurent Gatto <laurent.gatto@gmail.com> (fino al 2023)
-* Michael Love <michaelisaiahlove@gmail.com> (fino al 2024)
-* Federico Marini <fede.maro@gmail.com> (fino al 2024)
-* Sowmya Parthiban <sowmyaparthiban@gmail.com> (fino al 2024)
-* Anna Quaglieri <anna.quaglieri16@gmail.com> (fino al 2024)
-* Johannes Rainer <johannes.rainer@eurac.edu> (fino al 2023)
-* Lluís Revilla <lluis.revilla@gmail.com> (fino al 2024)
-* Charlotte Soneson <charlottesoneson@gmail.com> (fino al 2023)
+* (Presidente) Saskia Freytag sas.freytag@gmail.com (until 2023) she/her [EN, DE]
+* Laurent Gatto laurent.gatto@gmail.com (until 2023) he/him [FR, EN] (TAB, European Bioconductor Society)
+* Martín Dávila-Cervantes aldntmi@gmail.com (until 2025) he/him [EN, FR, ES] 
+* Federica Gazzelloni fede.gazzelloni@gmail.com (until 2025) she/her [IT,EN]
+* Gerry Tonkin-Hill g.tonkinhill@gmail.com (until 2025) he/him [EN] 
+* Michael Love michaelisaiahlove@gmail.com (until 2024) he/him [EN] (TAB)
+* Federico Marini fede.maro@gmail.com (until 2024) he/him [EN, IT, DE]
+* Sowmya Parthiban sowmyaparthiban@gmail.com (until 2024) she/her [EN, Tamil]
+* Shixiang Wang wangshx@shanghaitech.edu.cn (until 2025) he/him [CN,EN]
+* Anna Quaglieri anna.quaglieri16@gmail.com (until 2024) she/her [EN, IT]
+* Lluís Revilla lluis.revilla@gmail.com (until 2024) he/him [CAT, ES, EN]
+* Kevin Rue-Albrecht kevinrue67@gmail.com  (until 2024) he/him [FR, EN] (CAB)
+* (Ombusdperson) Michelle Savage michellesavage.galaxy@gmail.com (Galaxy project) she/her [ES,EN]
```

Comment:
sd -> ds

Reply:
non ho capito pardon, l'ho preso pari pari dalla versione americana.

Reply:
oh ok, got it, just changed in "mediatrice"